lms/templates/discussion/index.html

```diff
@@ -32,14 +33,14 @@
          data-course-id="${course_id}"
          data-course-name="${course.display_name_with_default}"
          data-user-info="${user_info}"
-         data-user-create-comment="${can_create_comment}"
-         data-user-create-subcomment="${can_create_subcomment}"
+         data-user-create-comment="${can_create_comment | n, dump_js_escaped_json}"
+         data-user-create-subcomment="${can_create_subcomment | n, dump_js_escaped_json}"
```

Comment:
I think the docs recommend just doing `${json.dumps(x)}` for this situation.

> On Aug 19, 2016, at 2:23 AM, Awais Jibran <notifications@github.com> wrote:
> 
> In lms/templates/discussion/index.html:
> 
> > @@ -32,14 +33,14 @@
> >           data-course-id="${course_id}"
> >           data-course-name="${course.display_name_with_default}"
> >           data-user-info="${user_info}"
> > -         data-user-create-comment="${can_create_comment}"
> > -         data-user-create-subcomment="${can_create_subcomment}"
> > +         data-user-create-comment="${can_create_comment | n, dump_js_escaped_json}"
> > +         data-user-create-subcomment="${can_create_subcomment | n, dump_js_escaped_json}"
> @robrap The problem is the Bool variables from python are being converted to a string. 
> True => "True" 
> If this should be reverted, what solution you would recommend? 
> Looping in @dianakhuang
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>